torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -49,7 +49,15 @@ std::shared_ptr<FutureMessage> RequestCallbackImpl::processRpc(
 
       // sc is only alive within this block, use reference to avoid copy
       auto& stack = scriptCall.stackRef();
-      scriptCall.op()->getOperation()(stack);
+      at::IValue res;
+      if (scriptCall.hasOp()) {
```

Comment:
can you provide the ScriptCall class a callable overload operator, and wrap these logics in there? This might clean the logic and apis (i.e. `hasOp` and `hasQualifiedName` is confusing, we could just run the call directly).

Reply:
will do

Reply:
tried to move the logic to script call class, then the script call class will have python dependency, this results in that all the files depending on script call class have to move from C++ torch lib to python torch libs. I feel it is a little overkilled. 

So still keeping the logic here